### PR TITLE
- fixes #8149: adds link to error response

### LINKED ISSUE #8149: != 200
[Enter feedback here]
What about the other cases (!= 200)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f276ed76-e7ad-6fca-5d10-64d88960fd77
* Version Independent ID: 33e996a2-c6a9-631a-56eb-7d7da548d2ef
* Content: [Update subscription - Microsoft Graph v1.0](https://docs.microsoft.com/en-us/graph/api/subscription-update?view=graph-rest-1.0&tabs=http#http-request)
* Content Source: [api-reference/v1.0/api/subscription-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/v1.0/api/subscription-update.md)
* Product: ****
* Technology: **microsoft-graph**
* GitHub Login: @baywet
* Microsoft Alias: **MSGraphDocsVteam**

### PR DESCRIPTION
**Pull request by baywet**

### CODE REVIEW

File: api-reference/beta/api/subscription-delete.md [hunk 1]
```diff
@@ -108,6 +108,13 @@ Here is an example of the response.
 HTTP/1.1 204 No Content
 ```
 
+## Error responses
```
Comment (Lauragra) [inline api-reference/beta/api/subscription-delete.md:111]: Global: This is a non-standard H2 heading for API topics. I recommend that you add the following sentence to the ## Response H2: "For details about how errors are returned, see [Error responses][error-response].

Comment (baywet) [reply api-reference/beta/api/subscription-delete.md]: thanks, fixed!

File: api-reference/beta/api/subscription-delete.md [hunk 2]
```diff
@@ -108,6 +108,11 @@ Here is an example of the response.
 HTTP/1.1 204 No Content
 ```
 
+For details about how errors are returned, see [Error Responses][error-response] for more info about
```
Comment (Lauragra) [inline api-reference/beta/api/subscription-delete.md:111]: Actually, I would put this in the ## Response section (the H2, not in the response example). Also, you just need a period after [Error responses], you can delete the rest of that text. And in the link text, "responses" should be lowercase.

Comment (baywet) [reply api-reference/beta/api/subscription-delete.md]: fixed!

**Pull request merged**